src/Interpreters/Context.cpp

```diff
@@ -2902,6 +2902,12 @@ BackgroundTaskSchedulingSettings Context::getBackgroundMoveTaskSchedulingSetting
     return task_settings;
 }
 
+size_t Context::getMaxPostponeTimeForFailedMutations() const
+{
+    const auto & config = getConfigRef();
+    return config.getUInt("max_postpone_time_for_failed_mutations", 0ull);
+}
```

Comment:
You can add this setting [here](https://github.com/ClickHouse/ClickHouse/blob/ee5d8c0a27a7d8d6cce0827d8761c290779b97e5/src/Core/Settings.h#L858)
and query it like `getSettingsRef().max_postpone_time...`